tests/publishdependency.targets

```diff
@@ -58,4 +90,147 @@
     </Copy>
   </Target>
 
+  <Target Name="CopyNonWindowsDependecyToCoreRoot"
```

Comment:
I believe you can pass in with '@' symbol would ensure the list can be passed in as a property and then use '%' inside to batch operate.